templates/dex-k8s-authenticator.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
 spec:
   kubernetes:
     minSupportedVersion: v1.14.0
+  cloudProvider:
+    all: disabled
```

Comment:
all disabled ? I don't get it

Reply:
I'll add more documentation in the kubeaddons library.